js_modules/dagster-ui/packages/ui-core/src/assets/AssetsCatalogTable.tsx

```diff
@@ -87,10 +91,22 @@ export const AssetsCatalogTable = ({
 }: AssetCatalogTableProps) => {
   const [view, setView] = useAssetView();
   const [search, setSearch] = useQueryPersistedState<string | undefined>({queryKey: 'q'});
-  const [searchGroups, setSearchGroups] = useQueryPersistedState<AssetGroupSelector[]>({
-    queryKey: 'g',
-    decode: (qs) => (qs.groups ? JSON.parse(qs.groups) : []),
-    encode: (groups) => ({groups: groups.length ? JSON.stringify(groups) : undefined}),
+
+  const [filters, setFilters] = useQueryPersistedState<{
```

Comment:
This seems great for now but I wonder if there's a generic version of this that calls JSON.stringify on all the `non-default-value` values in the object?